lib/Stickersheet.js

```diff
@@ -47,18 +47,35 @@ var Stickersheet = module.exports = function(trixel, options) {
     };
     
     var trixelWHratio = trixelViewBox.width / trixelViewBox.height;
-    
-    var trixelWidth = pageWidth * 0.3622; // Best guess based on mockup
+    var trixelWidth = pageWidth * 0.366; // Best guess based on mockup
     var trixelHeight = trixelWidth / trixelWHratio;
     
     function dupTrixel() {
-        // lodash's _.clone() method returns empty object if given a DOM node,
-        // even though these fake DOM nodes are of type Object so we do this.
-        var dupTrxl = document.createElement("svg");
-        dupTrxl.innerHTML = trixel.element.innerHTML;
-        dupTrxl.setAttribute("viewBox", trixel.element.getAttribute("viewBox"));
-        dupTrxl.setAttribute("width", trixelWidth);
-        return dupTrxl;
+
+        // Bleed == 1/16"
+        // TrixelWidth at 0.366% of 9" == 3.294"
+        // Inner trixel size is 3.294" - 0.0625" == 3.2315"
+        // So inner trixel size is 98.102611% of the outer trixel size
+
+        var dupTrxlInner = document.createElement("svg");
+        // Outer is for the bleed space
+        var dupTrxlOuter = document.createElement("svg");
+        var dupTrxlGroup = document.createElement("svg");
+
+        dupTrxlInner.innerHTML = trixel.element.innerHTML;
+        dupTrxlOuter.innerHTML = trixel.element.innerHTML;
+
+        dupTrxlInner.setAttribute("viewBox", trixel.element.getAttribute("viewBox"));
+        dupTrxlOuter.setAttribute("viewBox", trixel.element.getAttribute("viewBox"));
+
+        var innerTrxlScale = 0.98102611;
```

Comment:
Not sure about the specs on the stickers.  I just wanted to make sure the bleed indeed is going outside the cut line.  Maybe @semateos, Justin Varberakis, or @sefrankel would have word on that.